salt/edx/files/nginx_static_assets.j2

```diff
@@ -26,7 +26,9 @@
         expires epoch;
     }
 
+{% if salt.grains.get('business_unit') == 'mitxpro' %}
```

Comment:
I believe this should be `residential` as opposed to `mitxpro` as we only want that redirect to apply to those instances.